digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/GeometryTransform.scala

```diff
@@ -157,5 +158,34 @@ class GeometryTransform(roadAddressService: RoadAddressService) {
 
     (address, roadSide )
   }
+
+  /**
+    * @param roadLinkLength Length of road link
+    * @param addrSideCode SideCode of the road address for link, i.e. which way road address grows relative to digitizing direction
+    * @param addrStartMValue Road address distance from road link's start point (Point determined by road address growth direction NOT digitizing direction)
+    * @param addrEndMValue Road address distance from road link's end point (Point determined by road address growth direction NOT digitizing direction)
+    * @param assetStartMValue Start M-Value on road link for cut asset
+    * @param assetEndMValue End M-Value on road link for cut asset
+    * @return Road address start and end distance on cut asset
+    */
+  def getAddressMValuesForCutAssets(roadLinkLength: Double, addrSideCode: SideCode, addrStartMValue: Int, addrEndMValue: Int,
```

Comment:
Hyym en nyt tiedä kuuluistko tää olla täälä vai ennemin vaikkapa Roaddressutil tai jossakin vastaavassa kuntässä ei tehdä geometria muunnoksia.

Reply:
rivillä 74 oleva roadaddress case class vois kul kuulua paremmin tonne https://github.com/finnishtransportagency/digiroad2/tree/master/digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2 ja sitten Companion object erilaisia roadaddress metodeja

Reply:
GeometryTransform sinäänsä oikea paikka, kun katsoo mitä sieltä löytyy jo valmiiksi. GeometryTransform luokan voisi nimetä kyllä kuvaamaan sen vastuita paremmin, niin ei tarvitsisi tehdä isompaa refaktorointia

Reply:
Tää tais olla noihin viitekehysmuunnoksen logiikkoihin ja prosesseihin.  getAddressMValuesForCutAssets ei musta oikein kyl liity viitekehysmuunnoksiin.
 
Mikä  olis sun mielikuva tän tiedoston vastuista?

Reply:
Itse näkisin geometryTransform vastuuna tieosoitteiden sovittamisen lineaarilokaatioille (tielinkki, viivamainen asset)

Reply:
Ei tää taida tossa ajatuksessa oikein sopia tänne kun tässä ei enään soviteta tieosoiteta geometriaan kommentien perusteella se on jo tehty. Nyt vaan katsotaan leikatulle m-arvoja. 

Kaikki tuon tarvitsemat tiedot on tielinkisssä joten tuo ehkä kuuluisi ennemmin RoadLink class yhdeksi apumetodiksi ehkä.

Reply:
Hmm, ehkäpä selkein olisi jos tekisi uuden luokan RoadAddressUtils, jonne tunkisi noita tieosoitteiden apumetodeja. Siirtäisi sitten myös tuolta GeometryTransform.scala tiedostosta nuo tieosoitteisiin liittyvät luokat sinne.

Reply:
joo vois olla parempi

Reply:
Ei silleen utils metodit on kyl tavallaan sellainen placeholder kunne keksitään minkä class sisälle ne kuuluisi. Ehkä vaan RoadAddress class riittää.

Reply:
Tehdään näin